doc-content/enterprise-only/openshift/dashbuilder-standalone-proc.adoc

```diff
@@ -39,7 +48,10 @@ spec:
 ----
 
 +
-NOTE: You can add additional {KIE_SERVER} DataSets by repeating this step.
+[NOTE]
+====
+You can add additional {KIE_SERVER} data sets by repeating this step.
```

Comment:
```suggestion
You can add an additional {KIE_SERVER} data sets by repeating this step.
```
Can we mention the step number because **this step** slightly misleading?